indexer/ftypes_matcher.cpp

```diff
@@ -693,8 +693,8 @@ class UKRoadShieldParser : public SimpleRoadShieldParser
 {
 public:
   UKRoadShieldParser(std::string const & baseRoadNumber)
-    : SimpleRoadShieldParser(baseRoadNumber, {{'M', RoadShieldType::UK_Motorway},
-                                              {'A', RoadShieldType::UK_Highway}})
+    : SimpleRoadShieldParser(
```

Comment:
ftypes_matcher is devoted to BaseChecker and its subclasses only, moreover, quite specific subclasses of BaseChecker should not be here.